content/docs/reference/dns.mdx

```diff
@@ -0,0 +1,303 @@
+---
+id: dns
+title: DNS Settings
+description: DNS Settings
+keywords:
+  - reference
+  - DNS Lookup Family
+  - DNS
+pagination_prev: null
+pagination_next: null
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# DNS
+
+This reference covers all of Pomerium's **DNS Settings**:
+
+- [Failure Refresh Rate](#failure-refresh-rate)
+- [Lookup Family](#lookup-family)
+- [Query Timeout](#query-timeout)
+- [Query Tries](#query-tries)
+- [Refresh Rate](#refresh-rate)
+- [UDP Max Queries](#udp-max-queries)
+- [Use TCP](#use-tcp)
+
+## Failure Refresh Rate
+
+**Failure Refresh Rate** sets the DNS refresh rate when requests are failing.
+
+### How to Configure
+
+<Tabs>
+<TabItem value="Core" label="Core">
+
+| **Config file keys** | **Environment variables** | **Type** | **Default** |
+| :-- | :-- | :-- | :-- |
+| `dns_failure_refresh_rate` | `DNS_FAILURE_REFRESH_RATE` | `string` ([Go Duration](https://golang.org/pkg/time/#Duration.String) formatting) |  |
```

Comment:
The default is whatever Refresh Rate is, right?
https://www.envoyproxy.io/docs/envoy/v1.34.7/api-v3/extensions/clusters/dns/v3/dns_cluster.proto.html#extensions-clusters-dns-v3-dnscluster